packages/core/src/app/address/googleAutocomplete/GoogleAutocomplete.test.tsx

```diff
@@ -0,0 +1,33 @@
+import '@testing-library/jest-dom';
```

Comment:
The purpose of `import '@testing-library/jest-dom';` in `jest-setup.ts` is to avoid importing it in every single test file. Could you please check what is missing there?

Reply:
I didn't see any error after removing this line in both IDE and terminal.
<img width="500" alt="image" src="https://github.com/bigcommerce/checkout-js/assets/88361607/4c63c865-7d75-49bb-9148-37a6d81c9f65">

Reply:
@bc-peng  I get this kind of warnings when I remove that line.
<img width="1157" alt="Screenshot 2023-09-15 at 11 23 17 am" src="https://github.com/bigcommerce/checkout-js/assets/141614330/aac21f86-748c-4b89-97b9-4b6b05dd7a92">